website/content/docs/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,31 @@
+---
+layout: docs
+page_title: 1.14.0
+description: |-
+  This page contains release notes for Vault 1.14.0
+---
+
+# Vault 1.14.0 Release Notes
```

Comment:
Does it even make sense to have 1.14 docs in the 1.13 branch 🤔. I guess it's no harm just doesn't seem obvious we'd want that. Is it just too much effort to maintain older docs without newer updates that don't apply given the way our docs are presented?

Reply:
No, you're right, I think I resolved the merge incorrectly (meant to keep it deleted on this branch.)